code/modules/surgery/surgery.dm

```diff
@@ -25,7 +25,7 @@ GLOBAL_LIST_INIT(surgery_steps, init_surgery())
 	var/list/allowed_species = null //List of names referencing species that this step applies to.
 	var/list/disallowed_species = null
 
-
+	var/surgery_skill_required = SKILL_SURGERY_PROFESSIONAL
```

Comment:
autodoc

Reply:
What do you mean by autodoc?

Reply:
You use /// to comment what the variable does on the line above the variable.
Auto-documentation

Reply:
Oh! Automatice documentation! I thought you meant like the medbay machine autodoctor. I'm stupid.